backup/setup/migrations.json

```diff
@@ -1,183 +1,196 @@
+		},
+		"state": "up",
+		"name": "lessonIsCopyFrom",
+		"createdAt": "2019-07-11T07:44:37.148Z",
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d2ee2a8526cce83a8834d4a"
+		},
+		"state": "up",
+		"name": "addStartAndEndDateToYearCollection",
+		"createdAt": {
+			"$date": "2019-07-17T08:12:18.480Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d3ef5f4bd2cdb2075fc7e54"
+		},
+		"state": "up",
+		"name": "replace_grandlevel_id_with_number",
+		"createdAt": "2019-07-29T09:57:38.131Z",
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d3f03bf50c3d44bb83b93f8"
+		},
+		"state": "up",
+		"name": "migrateSchoolsTo2019-20",
+		"createdAt": {
+			"$date": "2019-07-29T14:33:35.456Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d442946c6c8392ba30b96a0"
+		},
+		"state": "up",
+		"name": "schoolyear_20_21_added",
+		"createdAt": {
+			"$date": "2019-08-02T12:15:02.584Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d4858fc8f66f50964686467"
+		},
+		"state": "up",
+		"name": "addToolPermissionsToSuperheroRole",
+		"createdAt": {
+			"$date": "2019-08-05T16:15:12.225Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d552b8bc1658a0682fc834a"
+		},
+		"state": "up",
+		"name": "set schoolspecific documentbasedir for instance open",
+		"createdAt": {
+			"$date": "2019-08-15T09:34:59.243Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d52a066a595714c04d763fb"
+		},
+		"state": "up",
+		"name": "AddClassAdministrationPermissionToAdminRole",
+		"createdAt": {
+			"$date": "2019-08-13T11:35:02.253Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d651f8f5e33068c88d5bccd"
+		},
+		"state": "up",
+		"name": "add_help_documents",
+		"createdAt": {
+			"$date": "2019-08-20T09:39:23.302Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d651f8f5e33068c88d5bcce"
+		},
+		"state": "up",
+		"name": "removeRcPws",
+		"createdAt": {
+			"$date": "2019-08-27T12:06:32.917Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d820e442d6f1435214b2d96"
+		},
+		"state": "up",
+		"name": "remove-research-and-dateofresearchconsent",
+		"createdAt": {
+			"$date": "2019-09-17T14:43:38.288Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d6fd5e8353b8668562740f3"
+		},
+		"state": "up",
+		"name": "addNewsCreateToAdmin",
+		"createdAt": {
+			"$date": "2019-09-04T15:19:04.053Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d80a25a92de001d94db5046"
+		},
+		"state": "up",
+		"name": "addUserDeletePermissions",
+		"createdAt": {
+			"$date": "2019-09-17T09:07:38.792Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5d91b525fee68c5c70d99de1"
+		},
+		"state": "up",
+		"name": "removeUnusedTeamAttributes",
+		"createdAt": {
+			"$date": "2019-09-30T07:46:22.613Z"
+		},
+		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "5da5bb0bc279258e75b1b510"
+		},
+		"state": "up",
+		"name": "addInsightsPermissionToAdminTeacher",
+		"createdAt": {
+			"$date": "2019-10-15T10:59:34.051Z"
+		},
+		"__v": 0
+	}
+]
```

Comment:
![image](https://user-images.githubusercontent.com/4550014/67300803-dab1f400-f4ee-11e9-9fe0-d1150b3ff02f.png)
set this :)

Reply:
done